integration/stacks_test.go

```diff
@@ -30,9 +31,11 @@ import (
 )
 
 const (
-	stackGitRepo   = "git@github.com:okteto/stacks-getting-started.git"
-	stackGitFolder = "stacks-getting-started"
-	stackManifest  = "okteto-stack.yml"
+	stackGitRepo     = "git@github.com:okteto/stacks-getting-started.git"
+	stackGitFolder   = "stacks-getting-started"
+	stackManifest    = "okteto-stack.yml"
+	composeGitRepo   = "git@github.com:okteto/flask-producer-consumer.git"
+	composeGitFolder = "flask-producer-consumer"
 )
 
 func TestStacks(t *testing.T) {
```

Comment:
do you think is worth unifying both tests to save time? the integration job already takes a long time.

Reply:
If I unify them I will need to modify the test to cover all the things that we support now in compose

Reply:
I will do two separate test. One for compose checking that is working and the other to test our own fields like endpoints